reference/carts.v3.yml

```diff
@@ -2330,35 +2348,32 @@ components:
                       expiresDate:
                         type: integer
                         example: 0
-                        description: Returns 0 if a expiration date is not set
+                        description: Returns 0 if no expiration date has been set.
                       totalDiscount:
                         type: number
                         example: 4.19
-                        description: Total discount amount off cart
+                        description: Total amount discounted off the entire cart.
                 required:
                   - coupon_code
         discount_amount:
           type: number
-          description: The total value of all discounts applied to this item. This includes coupons and cart level discounts
+          description: The total value of all discounts applied to this item. This includes coupons and cart-level discounts.
           example: 4
         coupon_amount:
           type: number
           description: The total value of all coupons applied to this item.
-        original_price:
-          type: number
-          description: An item’s original price is the same as the product default price in the admin panel.
         list_price:
           type: number
-          description: 'The net item price before discounts and coupons. BigCommerce derives an item’s list price from the product default price or, if applicable, the sale price configured in the admin panel.'
+          description: 'List price of the item, as quoted by the manufacturer/distributor.'
```

Comment:
@bc-traciporter this looks like a different version of the same sentence that confused me previously; when you reach a resolution, match them.

Reply:
@slsriehl This is the same as the previous ones. The definitions are the previous ones.